src/data/devtools.yaml

```diff
@@ -1,4 +1,45 @@
 name: Firefox - Developer Tools
-summary: DevTools is tools provided by Firefox for developers to inspect Web code
+summary: These are the tools provided by Firefox for developers and designers to inspect, debug and modify Web code.
+introduction: |
+  ## About DevTools
+
+  DevTools is a toolbox, built-in to Firefox, for developers and designers to inspect, debug and modify Web code.
+
+  You can learn about using DevTools by reading its [documentation](https://developer.mozilla.org/en-US/docs/Tools).
+ 
+  DevTools is written in Javascript, HTML and CSS and has lots of opportunities to help out.
+
+  ## Who Works on Taskcluster?
+  
+  A large community of volunteers throughout the world report bugs, test DevTools, submit code changes and localize the UI.
+  The team assigned to work on DevTools at Mozilla is about 20 people, distributed around Europe, Asia and North America.
+  You will likely see the names of team members listed as mentors in the bugs and issues on this site.
+  We are always excited to meet new Mozillians!
+  
+  ## How Do I Get Started?
+  
+  If you intend to work on a bug (in [Bugzilla](https://bugzilla.mozilla.org)) or an issue (in [GitHub](https://github.com)), make sure you have an account on those sites.
```

Comment:
We can track github issue too.. Do you want to add it? here is the example https://github.com/mozilla-frontend-infra/codetribute#best-practices